Formula/aom.rb

```diff
@@ -28,7 +29,8 @@ def install
                                     "-DENABLE_EXAMPLES=on",
                                     "-DENABLE_TESTDATA=off",
                                     "-DENABLE_TESTS=off",
-                                    "-DENABLE_TOOLS=off"])
+                                    "-DENABLE_TOOLS=off",
+                                    "-DBUILD_SHARED_LIBS=1"])
```

Comment:
```suggestion
                                    "-DBUILD_SHARED_LIBS=on"])
```
Should this be like the rest?